Demo/Sources/Views/PlaybackSlider.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import Player
+import SwiftUI
+
+#if os(iOS)
+struct PlaybackSlider: View {
+    let progressTracker: ProgressTracker
+    let minimumValueText: String?
+    let maximumValueText: String?
+
+    @Binding private var value: Float
+    @State private var valueWidth: CGFloat = 0
+    @State private var previousValueWidth: CGFloat = 0
+
+    @Binding private var buffer: Float
+    @State private var bufferWidth: CGFloat = 0
+
+    @State private var isDragging = false {
+        didSet {
+            progressTracker.isInteracting = isDragging
+        }
+    }
+
+    var body: some View {
+        HStack {
+            text(minimumValueText)
+            progressBar(valueWidth: valueWidth, bufferWidth: bufferWidth)
+            text(maximumValueText)
+        }
+        .frame(maxWidth: .infinity, minHeight: 10, maxHeight: 10)
+    }
+
+    init(progressTracker: ProgressTracker, minimumValueText: String? = nil, maximumValueText: String? = nil) {
```

Comment:
`minimumValueLabel` and `maximumValueLabel` parameters would provide API consistency with the native `Slider`, which would be quite nice IMHO. `EmptyView` specializations could then be introduced to implement optionality.